app/javascript/packages/webauthn/is-expected-error.spec.ts

```diff
@@ -10,7 +10,7 @@ describe('isExpectedWebauthnError', () => {
   });
 
   it('returns false for instance of DOMException of an unexpected name', () => {
-    const error = new DOMException('', 'UnknownError');
+    const error = new DOMException('', 'GeneralError');
```

Comment:
It probably doesn't make much a difference, but this is another one where we're updating code unrelated to the ticket, and probably should be left as-is.

```suggestion
    const error = new DOMException('', 'UnknownError');
```